### DISCUSSION TITLE
How to run opencanary as noprivileged user

### DISCUSSION THREAD

**Discussion by hitjackma** [thinkst/opencanary#273] [Feature Requests]
Hi， It's possible to run opencanary as normal user, not root, like how nginx do run as nginx?

**Comment by jayjb** [comment]
Hi @hitjackma,

It may be possible. The reason we default to it is because the modules such as `portscan` (which uses iptables) require `sudo` privileges. I think it is a reasonable question though. Perhaps we could do some categorising of services that are available on 'less than root' privileges.
